lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -118,15 +180,14 @@ class window.Turbolinks
     triggerEvent 'page:receive'
     options.updatePushState ?= true
 
-    if doc = processResponse(xhr, options.partialReplace)
-      reflectNewUrl url if options.updatePushState
-      nodes = changePage(extractTitleAndBody(doc)..., options)
-      reflectRedirectedUrl(xhr) if options.updatePushState
+    doc = processResponse(xhr, options.partialReplace)
```

Comment:
Having Turbograft depend on the format of asset names/fingerprints couples it tightly with sprockets. It's better to be explicit by defining a unique ID for each asset.

One example where depending on sprockets fails is when we rename a bundle (let's say `inventory_transfers_bundle.js` to `transfers_bundle.js`). Parsing the name from the `src` will cause TG to treat this as a brand new asset and we will have the duplicate event listener problem from before. However using a unique ID separate from the `src` let's us tell TG that it is still the same bundle even if the name changes.